Monika After Story/game/pong.rpy

```diff
@@ -430,28 +604,498 @@ init -1 python in mas_pong:
+    if monika_asks_to_go_easy and ball_paddle_bounces == 1:
+        m 2lssdrb "Ahaha.."
+        m 1hsb "This isn't exactly what I had in mind."
+        m 4esb "However, I do appreciate the gesture~"
+        $ monika_asks_to_go_easy = False
+        
+        
+    #Player lets Monika win after being asked to go easy on her without hitting the ball too much
+    if monika_asks_to_go_easy and ball_paddle_bounces <= 9:
+        m 1esb "Yay, I won!"
+        m 5ekbfa "Thanks, [player]!"
+        m 5hubfb "You are awesome~"
+        $ monika_asks_to_go_easy = False
+        
+        
+    #The player fails to hit the first ball
+    elif ball_paddle_bounces == 1:
+        
+        #One time
+        if instant_loss_streak_counter == 1:
+            m 5hub "Ahaha, how did you miss that?"#
+            
+            
+        #Two times
+        elif instant_loss_streak_counter == 2:
+            m 2tsb "[player], you missed again!"#
+            
+            
+        #Three times
+        elif instant_loss_streak_counter == 3:
+            m 2tub "[player]!"#
+            
+            if persistent._mas_pm_ever_let_monika_win_on_purpose:
+                m 5rub "Are you letting me win on purpose again?"#
+            else:
+                m 5eub "Are you letting me win on purpose?"#
+                
+            menu:
+                "Yes":
+                    m 1hub "You are so cute, [player]!"#
+                    m 5hua "Thank you for letting me win~"
+                    if persistent._mas_pm_ever_let_monika_win_on_purpose:
+                        m 1ekbsa "But you know I wouldn't mind losing against you."#
+                        m 5hua "I like to see you win just as much as you like to see me win~"
+                    else:
+                        m 1eka "I wouldn't mind losing against you, though."#
+                    $ player_lets_monika_win_on_purpose = True
+                    $ persistent._mas_pm_ever_let_monika_win_on_purpose = True
+                "No": 
+                    if persistent._mas_pm_ever_let_monika_win_on_purpose:
+                        m 1ttu "Are you sure?"
+                        menu:
+                            "Yes": 
+                                m 1hua "Alright."#
+                                m 2ekc "I'm sorry for assuming then..."
+                                m 3ekb "In case you can't concentrate, maybe you should take a break."
+                                m 4eka "Drinking a glass of water can also help."
+                                $ player_lets_monika_win_on_purpose = False
+                            "No":
+                                m 1rfu "[player]."#
+                                m 2eub "Stop teasing me!"
+                                $ player_lets_monika_win_on_purpose = True
+                                $ lose_on_purpose = True
+                        
+                    else:
+                        m 2ekd "I'm sorry for assuming..."#
```

Comment:
That is exactly the point.
They are not the same and are not supposed to be the same.

Reply:
```renpy
m 2ekc "I'm sorry for assuming then..."
m 3ekb "In case you can't concentrate, maybe you should take a break."
m 4eka "Drinking a glass of water can also help."
```
vs.

```renpy
m 2ekd "I'm sorry for assuming..."#
m 3eka "If you can't concentrate, maybe you should take a break."
m 4eka "Drinking a glass of water can also help."
```
Sure, the first line is different, and upon looking at it again, it might be fine w/o the `then` as well. I still feel the expressions in the first one look better though, hence why I say it.

Reply:
Why does it matter how it looks?
This is not a beauty-contest, Monika is expressing herself.
Also, you may like a version better but this doesn't mean everyone does.